lib/internal/errors.js

```diff
@@ -1207,7 +1207,9 @@ E('ERR_UNHANDLED_ERROR',
 E('ERR_UNKNOWN_BUILTIN_MODULE', 'No such built-in module: %s', Error);
 E('ERR_UNKNOWN_CREDENTIAL', '%s identifier does not exist: %s', Error);
 E('ERR_UNKNOWN_ENCODING', 'Unknown encoding: %s', TypeError);
-E('ERR_UNKNOWN_FILE_EXTENSION', 'Unknown file extension: %s', TypeError);
+E('ERR_UNKNOWN_FILE_EXTENSION',
+  'Unknown file extension \'%s\' for %s imported from %s',
```

Comment:
Nit:

```suggestion
  "Unknown file extension '%s' for %s imported from %s",
```

Reply:
@BridgeAR For the sake of uniformity, how about I invert the quotation marks instead: `'Unknown file extension "%s" for %s imported from %s'`? There are many like that (L896, L901, etc) but none with outer double quotes.

Reply:
I am good both ways. We do not have any convention around this as far as I know. I just prefer the quotes to be different so that it's easier to read the text.

Reply:
Sounds good, made the changes